src/scenes/App/components/TermsAndConditions/index.jsx

```diff
@@ -0,0 +1,274 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
+import { connect } from 'react-redux';
+import { withStyles } from 'material-ui/styles';
+import Paper from 'material-ui/Paper';
+import Typography from 'material-ui/Typography';
+import { FormattedMessage } from 'react-intl';
+import Grid from 'material-ui/Grid';
+import { FormGroup, FormControlLabel } from 'material-ui/Form';
+import Checkbox from 'material-ui/Checkbox';
+import Button from 'material-ui/Button';
+
```

Comment:
Let's combine some of these import statements.
```
import { Paper, Typography, Grid, Checkbox, Button, withStyles } from 'material-ui';
```

Reply:
done